test/Tester/GrainServiceTests/GrainServiceTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Orleans.Runtime.Configuration;
+using Orleans.TestingHost;
+using TestExtensions;
+using UnitTests.GrainInterfaces;
+using Xunit;
+
+namespace Tester
+{
+    public class GrainServiceTests : TestClusterPerTest
+    {
+        public override TestCluster CreateTestCluster()
+        {
+            var options = new TestClusterOptions(1);
+            options.ClusterConfiguration.UseStartupType<GrainServiceStartup>();
+
+            return new TestCluster(options);
+        }
+
+        [Fact, TestCategory("BVT"), TestCategory("Functional"), TestCategory("GrainServices")]
+        public async Task SimpleInvokeGrainService()
+        {
+            // We need to get the Silo to create the GrainService instances and register them as SystemTargets.
```

Comment:
I need to find a way to bootstrap creating the GrainService SystemTargets in the Silo on Startup.